builder.go

```diff
@@ -34,6 +34,7 @@ git clone --depth 1 {{.CloneURL}} repo
 cd repo
 git fetch origin {{.Ref}}
 git checkout -qf {{.Head}}
+git submodule update --init
```

Comment:
I'd thought that I'd have to run `git submodule update` within that empty mount directory. So it can be run in the root directory of the source code tree!